client/packages/common/src/types/schema.ts

```diff
@@ -1999,6 +1999,26 @@ export type LocationSortInput = {
 
 export type LocationsResponse = LocationConnector;
 
+export enum LogLevelEnum {
```

Comment:
From the merge from develop since log changes haven't been implemented in fe yet

Reply:
You could have just committed the lines that are relevant for this PR (is this possible with the github client?) However, since this is auto generated it shouldn't be a problem, i.e. you can just re-generated if there is a merge problem...